sdk/eventhub/event-hubs/test/eventProcessor.spec.ts

```diff
@@ -96,4 +100,58 @@ describe("Event Processor", function(): void {
       isCloseCalled.should.equal(true);
     });
   });
+
+  describe("InMemory Partition Manager", function(): void {
+    const inMemoryPartitionManager = new InMemoryPartitionManager();
```

Comment:
I think you'll want a separate `InMemoryPartitionManager` for each test to prevent state leakage between tests.

You can do that pretty easily with a `beforeEach`:
```typescript
let inMemoryPartitionManager: InMemoryPartitionManager;
beforeEach(function(): Promise<void> {
  inMemoryPartitionManager = new InMemoryPartitionManager();
});
```